Blog Migration: Continuation of WIP PR #7144

tehut opened the pull request:
Continuation of WIP PR #7144 

Migration currently feature complete with the following issues remaining to be resolved:
Intended Fix:  Switch from official paginate gem to popular paginate-v2
      - inconsistent resolution of liquid tags by official paginate gem on aggregation page
      - had some trouble propagating changes that appeared on the index page.  This is part
         of the issue with the main paginate-gem and the primary reason to switch
Still missing:
- Documentation
- Codebase clean up
https://github.com/kubernetes/website/pull/7144
> NOTE: After opening the PR, please *un-check and re-check* the ["Allow edits from maintainers"](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) box so that maintainers can work on your patch and speed up the review process. This is a temporary workaround to address a known issue with GitHub.> 
>
> Please delete this note before submitting the pull request.

![Allow edits from maintainers checkbox](https://help.github.com/assets/images/help/pull_requests/allow-maintainers-to-make-edits-sidebar-checkbox.png)

tengqm commented:
/assign @chenopis

heckj commented:
👋 I tracked down the netlify failure (either above, or related to https://app.netlify.com/sites/kubernetes-io-master-staging/deploys/5a83dc1cfd0efa61de658997) - there's an issue with dependencies when using the jekyll plugins that were defined and added as a part of this work - specifically "jekyll-youtube". I'll circle back and see if I can help identify what versions are needed, as I was doing some emergency repair work on dependency updates recently to solve a build/deploy failure sequence yesterday

heckj commented:
/assign

heckj commented:
I snagged this and rebased it, resolving the netlify build problem. One gemfile needed to be updated, and the whole Gemfile.lock benefited from an update. You can see the delta comparing to https://github.com/kubernetes/website/pull/7416.

The short notes on how to do this:

    rvm use ruby-2.2.9
    bundle install
    bundle update

and then review the Gemfile and Gemfile.lock files for changes. I can make a PR back to tehut's branch, you can just cherry pick what you want from my branch, or you can replicate the steps - but it's important to rebase this off github.com/kubernetes/website master branch to also get the changes we made to resolve from jekyll build breaks yesterday

heckj commented:
It does look like you've enabled maintainers to update this branch, so there's also the option of me pushing a commit onto this branch if that would be easier...

tehut commented:
@heckj Thanks again and I'm sorry I didn't see the comments on this PR earlier!  I updated the PR with your commit and everything worked beautifully so feel free to close #7416.

@zacharysarah , et al I added a small fix to the sidebar between PRs and it looks like it fixed one thing and broke another.  Just wanted to document that I've seen the issue and will address it in my next PR

heckj commented:
/recheck
/approve

heckj commented:
@tehut the very last commit is set up with a different "author" on the commit, and the CLA bot is having a git about it, since it's not your github userid. Could you rebase & squash the commits and/or ammend that particular commit so that all the commits in this PR have a github ID associated with them?

tehut commented:
@zacharysarah  @heckj Sorry about the delay with my rebase.  It looks like everything but tide pool is passing now.

zacharysarah commented:
/lgtm
